src/MultiRobot.elm

```diff
@@ -0,0 +1,67 @@
+module MultiRobot exposing (main)
```

Comment:
`main` isn't very useful outside.
Export `Msg`, `Model`, `init`, `view`, `update`, `subscriptions` instead, so that you can embed this module in a bigger one, just like `Robot.elm` is embedded here.